src/dataflow/dataflow.sass

```diff
@@ -167,6 +167,10 @@ $list-item-scale: 0.1
               background-color: $gray-light-hover
             &:active
               background-color: $gray-light-click
+            &.primary-doc
+              background-color: $gray-light-click
+            &.comparison-doc
+              background-color: $gray-light-click
```

Comment:
For now both colors are the same.  I added the two classes in case we want to style them differently in the future.